src/current/molt/molt-fetch.md

```diff
@@ -369,10 +372,71 @@ To cancel replication, enter `ctrl-c` to issue a `SIGTERM` signal. This returns
 `import-only` instructs MOLT Fetch to load the source data in the specified [cloud storage](#cloud-storage) or [local file server](#local-file-server) into the CockroachDB target.
 
 {% include_cached copy-clipboard.html %}
-~~~ 
+~~~ shell
 --mode import-only
 ~~~
 
+#### Fail back to source database
+
+{{site.data.alerts.callout_info}}
+`failback` mode supports MySQL 8.0 and later.
+{{site.data.alerts.end}}
+
+If you encounter issues after moving data to CockroachDB, you can use `failback` to replicate changes on CockroachDB back to the initial source database. In case you need to roll back the migration, this ensures that data is consistent on the source.
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+--mode failback
+~~~
+
+When running `molt fetch --mode failback`, `--source` is the CockroachDB connection string and `--target` is the connection string of the database from which you migrated. `--table-filter` specifies the tables to watch for change events. For example:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+--source 'postgresql://{username}:{password}@{host}:{port}/{database}'
+--target 'mysql://{username}:{password}@{protocol}({host}:{port})/{database}'
+--table-filter 'employees, payments'
+~~~
+
+`failback` mode creates a [CockroachDB changefeed]({% link {{ site.current_cloud_version }}/change-data-capture-overview.md %}) that passes change events from CockroachDB to a [webhook sink]({% link {{ site.current_cloud_version }}/changefeed-sinks.md %}#webhook-sink). However, if there is already a running CockroachDB changefeed with the same webhook sink URL (excluding query parameters) and [watched rows]({% link {{ site.current_cloud_version }}/changefeed-sinks.md %}), the existing changefeed is used for `failback`.
+
+When creating a new `failback` changefeed, MOLT Fetch specifies the following [`CREATE CHANGEFEED` parameters]({% link {{ site.current_cloud_version }}/create-changefeed.md %}#parameters):
+
+~~~ json
+{
+    "host": "localhost",
+    "port": 30004,
+    "sink_query_parameters": "insecure_tls_skip_verify=true",
+    "resolved": "1s",
+    "min_checkpoint_frequency": "1s",
+    "initial_scan": "no",
+    "webhook_sink_config": "{\"Flush\":{\"Bytes\":1048576}}"
+}
+~~~
+
+By default, MOLT Fetch specifies a local webhook sink (`"localhost"`) and an insecure sink connection (`"insecure_tls_skip_verify=true"`). This configuration is best suited for testing. A **secure sink connection is recommended** for production environments. For an example, see [Fail back securely from CockroachDB](#fail-back-securely-from-cockroachdb).
```

Comment:
@rohan-joshi @taroface @Jeremyyang920 there is a part of me wondering if we should make this secure by default and allow `--allow-tls-mode-disable` to dictate if we do this local setup or not. 

However, this means the defaults we set won't work unless you have this flag set. It's not as great a UX, but it is a secure default which seems to be important to note. Thoughts?

Reply:
I'm not the decider here, but it did feel counterintuitive to explain in docs that the default shouldn't be used in production.